README.md

```diff
@@ -125,10 +125,11 @@ All APIs and Models' documentation can be found at the [Generated client's READM
 
 ## Compatability
 
-|                   | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
-|-------------------|----------------|----------------|----------------|----------------|
-| java-client 0.1.0 | +              | +              | ✓              | -              |
-| java-client 0.2.0 | +              | +              | ✓              | -              |
+|                         | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|-------------------------|----------------|----------------|----------------|----------------|
+| java-client 0.1.0       | +              | +              | ✓              | -              |
+| java-client 0.2.0       | +              | +              | ✓              | -              |
+| java-client 1.0.0-beta1 | +              | +              | ✓              | -              |
```

Comment:
Should we try to update client for 1.8 before the release. It should be straightforward, right?

Reply:
Oh good idea - go for it!

Reply:
I'll just rebase this over the updated client when ready

Reply:
Sure, I can do it.